packages/broker/src/plugins/operator/MaintainTopologyService.ts

```diff
@@ -16,72 +13,49 @@ function toStreamIDSafe(input: string): StreamID | undefined {
     }
 }
 
-function singletonSet<T>(element: T): Set<T> {
-    return new Set<T>([element])
-}
-
 export class MaintainTopologyService {
     private readonly streamrClient: StreamrClient
-    private readonly operatorClient: OperatorClient
+    private readonly maintainTopologyHelper: MaintainTopologyHelper
     private readonly subscriptions = new Multimap<StreamID, Subscription>()
     private readonly synchronizer = new SetMembershipSynchronizer<StreamID>()
     private readonly concurrencyLimit = pLimit(1)
+    private readonly logger: Logger
```

Comment:
I would keep the static logger pattern as before. So
```ts
const logger = new Logger(module)
```
at the beginning of this file and then just use that. That way the log message have a clear module name in their statements, e.g.
```
INFO [2023-06-19T07:20:06.010] (MaintainTopologyService  ): Logging something
```